docs/templates.md

```diff
@@ -148,6 +148,15 @@ defined.
 * `.remove_suffix(needle: Template) -> String`: Removes the passed suffix, if present
 * `.substr(start: Integer, end: Integer) -> String`: Extract substring. Negative values count from the end.
 
+#### String literals
+
+String literals must be surrounded by double quotes (`"`). The following escape
+sequences starting with a backslash have their usual meaning: `\"`, `\\`, `\n`,
+`\r`, `\t`, `\0`. Other escape sequences are not supported. Any UTF-8 characters
+are allowed inside a string literal, with two exceptions: unescaped `"`-s and
+uses of `\` that don't form a valid escape sequence.
+
+
```

Comment:
nit: some markdown formatter told me to not use double blank lines

Reply:
Good catch, this was entirely unwarranted here. Are you saying you want to get rid of the other double-blank-lines-before-headings in general?

Markdown formatters are weird, they have different opinions and some can be configured extensively (e.g. to require double-blank-lines before some headings).